projects/plugins/jetpack/_inc/client/search/dashboard/module-control.scss

```diff
@@ -35,3 +40,42 @@ p.jp-search-search-toggle__explanation {
 	font-weight: 250;
 	font-size: 0.75rem;
 }
+
+.jp-form-search-cfa-buttons {
```

Comment:
Are we adhering to BEM standards here? If not, should we? (I'm not sure.)

Reply:
We definitely should. Addressed in 3cc319d72.